editors/code/src/commands.ts

```diff
@@ -481,12 +482,67 @@ export function viewItemTree(ctx: Ctx): Cmd {
 
 function crateGraph(ctx: Ctx, full: boolean): Cmd {
     return async () => {
-        const panel = vscode.window.createWebviewPanel("rust-analyzer.crate-graph", "rust-analyzer crate graph", vscode.ViewColumn.Two);
+        const nodeModulesPath = vscode.Uri.file(path.join(ctx.extensionPath, "node_modules"));
+
+        const panel = vscode.window.createWebviewPanel("rust-analyzer.crate-graph", "rust-analyzer crate graph", vscode.ViewColumn.Two, {
+            enableScripts: true,
+            retainContextWhenHidden: true,
+            localResourceRoots: [nodeModulesPath]
+        });
         const params = {
             full: full,
         };
-        const svg = await ctx.client.sendRequest(ra.viewCrateGraph, params);
-        panel.webview.html = svg;
+
+        const dot = await ctx.client.sendRequest(ra.viewCrateGraph, params);
+
+        const scripts = [
```

Comment:
This seems a bit overkill. Your choice, but I'd probably use `${nodeModulesPath}` directly in the HTML template below.

Reply:
You are right. I tried to simplify this in the new PR update. Thank you.